SingularityUI/app/views/task.coffee

```diff
@@ -61,4 +66,30 @@ class TaskView extends View
             setTimeout (=> @trigger 'refreshrequest'), 1000
 
 
+    executeCommand: (event) ->
+        event.preventDefault()
+        cmd = $("#cmd option:selected").text()
+        options = $('#cmd-option').val()
+        return if @$('button[type="submit"]').attr 'disabled'
```

Comment:
let's be more specific and reference this button using an ID, always better to be explicit